edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/IDeviceScopeApiClientProvider.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.CloudProxy
+{
+    using Azure.Devices.Edge.Util;
+
+    public interface IDeviceScopeApiClientProvider
```

Comment:
this interface should have 1 method CreateDeviceScopeClient() and implementation based on the feature if it is turned on or not.